src/mempool_prioritization.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2019-2020 Xenios SEZC
+// https://www.veriblock.org
+// Distributed under the MIT software license, see the accompanying
+// file LICENSE or http://www.opensource.org/licenses/mit-license.php.
+
+#include "veriblock/mempool_prioritization.hpp"
+
+namespace altintegration {
+
+bool isStronglyEquivalent(const VTB& vtb1,
```

Comment:
I think since you set asserts that containing blocks must exist on VBK tree, it would be better to move this function to VbkBlockTree (as member func)